lib/http2/connection.c

```diff
@@ -285,6 +287,11 @@ static void preserve_stream_scheduler(h2o_http2_conn_t *conn, h2o_http2_stream_t
     h2o_http2_scheduler_deactivate(&(*dst)->_scheduler);
 }
 
+void h2o_http2_conn_preserve_stream_scheduler(h2o_http2_conn_t *conn, h2o_http2_stream_t *src)
+{
+    preserve_stream_scheduler(conn, src);
+}
```

Comment:
I think we'd prefer promoting `preserve_stream_scheduler` to an external function by adding `h2o_http2_conn_` as the prefix.

I understand the intention to minimize the changes, but that comes with the complexity of adding a wrapper, which increases the maintenance cost in the long run.